common/lib/dependabot/clients/azure.rb

```diff
@@ -22,7 +22,7 @@ class TagsCreationForbidden < StandardError; end
 
       RETRYABLE_ERRORS = [InternalServerError, BadGateway, ServiceNotAvailable].freeze
 
-      MAX_PR_DESCRIPTION_LENGTH = 3999
+      PR_DESCRIPTION_MAX_LENGTH = 3999
```

Comment:
I'm confused... why is this both here and also in `common/lib/dependabot/pull_request_creator/azure.rb`??

I'm guessing it should be removed from this file?